code/datums/components/basic_mob_attack_telegraph.dm

```diff
@@ -62,11 +62,14 @@
 	if (!do_after(source, delay = telegraph_duration, target = target, timed_action_flags = IGNORE_TARGET_LOC_CHANGE, extra_checks = CALLBACK(source, TYPE_PROC_REF(/atom/movable, Adjacent), target), interaction_key = INTERACTION_BASIC_ATTACK_FORCEAST))
 		forget_target(target)
 		return
-	if (isnull(target)) // They got out of the way :(
+	if (isnull(target)) // They got out of the way, sad
```

Comment:
put the smiley back :(

Reply:
It breaks bracket pair coloring which is annoying